src/components/rmrk/Gallery/GalleryCard.vue

```diff
@@ -58,9 +59,11 @@ export default class GalleryCard extends Vue {
       const meta = await get(this.metadata);
       if (meta) {
         this.image = sanitizeIpfsUrl(meta.image)
+        this.title = meta.name
```

Comment:
Where do you use this property?

Reply:
In GalleryCard used as alternative text with the nft name